pom.xml

```diff
@@ -63,8 +63,8 @@
     </build>
     <dependencies>
         <dependency>
-            <groupId>javax.activation</groupId>
-            <artifactId>activation</artifactId>
+            <groupId>com.sun.activation</groupId>
```

Comment:
the newest version is available from https://search.maven.org/artifact/com.sun.activation/jakarta.activation/1.2.1/jar.